__tests__/app.test.js

```diff
@@ -107,4 +107,42 @@ describe('GET /api', () => {
 				});
 		});
 	});
+
+	describe('GET /api/articles/:article_id/comments', () => {
+		test('200: Responds with an array of comment objects for the input article id', () => {
+			return request(app)
+				.get('/api/articles/3/comments')
+				.expect(200)
+				.then(({ body: { comments } }) => {
+					expect(comments).toHaveLength(2);
+					expect(comments).toBeSortedBy('created_at', { descending: true });
+					comments.forEach((comment) => {
+						expect(comment).toMatchObject({
+							comment_id: expect.any(Number),
+							body: expect.any(String),
+							article_id: 3,
+							author: expect.any(String),
+							votes: expect.any(Number),
+							created_at: expect.any(String),
+						});
+					});
+				});
+		});
+		test('404: Responds with an error message when passed a valid url parameter but there is no content', () => {
```

Comment:
Good test. What about when the article with the given id exists, but there are no comments associated with it? What are you expecting to be returned? Should this be a `200` or a `404`?